src/main/java/com/segment/analytics/android/integrations/google/analytics/GoogleAnalyticsIntegration.java

```diff
@@ -180,31 +185,41 @@
     sendProductEvent(event, category, properties);
 
     if (COMPLETED_ORDER_PATTERN.matcher(event).matches()) {
+
+      // initialize instance of ScreenViewBuilder class
+      ScreenViewHitBuilder builder = new ScreenViewHitBuilder();
```

Comment:
Here's an example of a transaction event: https://developers.google.com/analytics/devguides/collection/android/v4/enhanced-ecommerce#measuring-transactions. It seems all their enhanced ecommerce events are associated with screen views in their examples. Although I could probably attach Completed Order, Product Viewed, etc., events to event hits, seen here: https://developers.google.com/analytics/devguides/collection/android/v4/events